libpod/kube.go

```diff
@@ -468,11 +468,28 @@ func generateKubeSecurityContext(c *Container) (*v1.SecurityContext, error) {
 		return nil, err
 	}
 
+	var selinuxOpts v1.SELinuxOptions
+	opts := strings.SplitN(c.config.Spec.Annotations[InspectAnnotationLabel], ":", 2)
+	if len(opts) == 2 {
+		switch opts[0] {
```

Comment:
I'm 99% sure that this is going to be set to `disabled` in almost every use, so we need to support that

Reply:
what should we do here if it's `disable`?

Reply:
That's an excellent question... Can Kube YAML not request SELinux be turned off? @rhatdan

Reply:
I actually don't think it can on the kube yaml level. I am pretty sure the only way to disable selinux in a pod is to give it privilege

Reply:
here's the object definition: https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.17/#selinuxoptions-v1-core, and per: https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.17/#podsecuritycontext-v1-core if one isn't specified the runtime allocates one (if it's privileged it's given spc_t, which effectively disables)

Reply:
can it be hardcoded `selinuxOpts.Type = "spec_t"` to type:spc_t?

Reply:
yeah that can be done